app/assets/stylesheets/application.css.scss

```diff
@@ -15,64 +15,69 @@
  *= require foundation_and_overrides
 
  */
+ body, h2, h4 {
+   background-color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Each selector in a comma sequence should be on its own line